### ISSUE TITLE
Please re-add the option to manually select all applicable explicit resistances.

### ISSUE THREAD

**Issue by flapanther33781** [SnosMe/awakened-poe-trade#483]
After the last update it seems the tool will only show the highest single resistance.  I want to be able to search for both the general "Total Elemental Resistance" as well as the specific rolls on the item, and your tool having the ability to add up those individual rolls was very handy because of items like this:

	--------
	+15% to Cold and Lightning Resistances (implicit)
	--------
	Adds 1 to 2 Physical Damage to Attacks
	+2 to maximum Energy Shield
	+54 to maximum Life
	+5% to all Elemental Resistances
	+45% to Lightning Resistance
	+14% to Fire and Cold Resistances (crafted)
	--------

Try adding up those resistances in your head.  Clearly it was faster with your tool doing those calculations.

The current pricing for a ring matching "Total Elemental Resistance" is only 2c but for the specific rolls is 10c.  Doing both searches gives me a more accurate idea of how to price my item.

**Comment by hejz0r** [comment]
@SnosMe as @flapanther33781 said - **bullshit** - if item has only 2 available resistances with closed suffix, how you change it with harvest craft to make it 3?

Items with x3 ele res are more valuable than x2 ele res with closed suff - now we cannot search properly

**Comment by SnosMe** [comment]
@hejz0r 

> Items with x3 ele res are more valuable

Having cold+30,fire+30,light+30 each is the same, as cold+45,fire+45. Can you answer me **why having x3 res on item is better for CHARACTER then x2**, if it's better than indeed you can request more price for such item?

If you really want to get extra money from item, this filter is doing it much better.

![image](https://user-images.githubusercontent.com/4292308/142278265-ceb81e1a-1463-4d08-b19c-94cb33048a7e.png)

> has only 2 available resistances with closed suffix, how you change it with harvest craft

You're so obsessed with x3 resistances that don't even notice that I'm talking about change res, not aug.

![image](https://user-images.githubusercontent.com/4292308/142278012-11ab23cf-8b31-469e-8f58-f11a99519791.png)

**Comment by hejz0r** [comment]
You should know - You removed QoL from your addon, thats all

**Comment by hejz0r** [comment]
and hell no, 2 x 45% is not the same as 3x 30% when pricing single item, your listing says 25ch is the cheapest option and i'm selling it for 80ch in 1h

**Comment by SnosMe** [comment]
> and hell no, 2 x 45% is not the same as 3x 30% when pricing single item, your listing says 25ch is the cheapest option and i'm selling it for 80ch in 1h

I'll quote for you again 
> You are just fooled by narrower number of results returned when you search for x3 res, get less results = price rises between items more quick. But it doesn't mean that x3 res is the actual reason for this.

> why having x3 res on item is better for CHARACTER then x2

**Comment by hejz0r** [comment]
just curious what qol will you delete next update? maybe go straight forward and abandon project if u dont want to listen simple users?

**Comment by hejz0r** [comment]
@SnosMe - and yes, im fooled of 3 res, because of simple matter - random people just pay more for 3 res if they see 3 good res on item, not every1 is a PRO to perfectly manage their resistances - and you are just an ignorant, you have removed an option which was ok and was not hurting anybody.  Don't know exactly what you expect? That people will flatter sayin "Oh thanks Snos for deleting resistance search" ^^ Addon you created is fucking good, dont know why are u cutting features. 

Even poeprices let you filter every resistance ...

**Comment by SnosMe** [comment]
> more for 3 res if they see 3 good res

I think it's applicable only for league start item, because most of the time you're searching for some specific stat, while resistances just a "bonus".

Like here can only be TWO res suffixes.

![image](https://user-images.githubusercontent.com/4292308/142283330-33010813-59c4-402b-be14-016e4c60d372.png)

It doesn't matter of which ele types they are because of Harvest.

**Comment by hejz0r** [comment]
@SnosMe - just remember, 90% of the market are items priced less than 1ex, people use your item to value random drop from maps ..

**Comment by flapanther33781** [comment]
Man, I love your tool, there are so many ways you are wrong about this specific thing that it's actually laughable.

> Can you answer me why having x3 res on item is better for CHARACTER then x2

Because there are many, many players who do not know what you know, and are willing to pay.  Period.  That's all we should have to say.  Why do you need more answers?

If you want the specific reason, it's because someone might we looking to upgrade an item without wanting (or maybe having the money) to upgrade ALL their items.  Say it's their belt.  They unequip their belt and see which resistances drop below 75%.  Maybe they need 15% cold, 20% fire, and 30% lightning to regain their cap.  When searching for their new belt they add all three mods in the GGG site to make sure their new item has those in addition to the other mods they want to search for.

For someone who doesn't know crafting (like I didn't for a long time) this is just how you search.  The search you describe is more complex and didn't even occur to me until a few leagues later when someone told me you could search for empty prefixes or suffixes.  Even now that I know this, I will sometimes search for an item that has all three mods (if I want that) and compare the price to ones with an open suffix because if I can get one that is perfectly rolled for only a little more that might be worth a few c to me. I can't believe you're fighting this so hard lol

> I think it's applicable only for league start rare items, after this point in league most of the time you're searching for some specific stat, while resistances just a "bonus".

No, it is only relevant for league start items for YOU and players like you who (a) know how to craft the item, and (b) are willing to.

Regarding (a): many people do not, or do not want to, and are willing to pay higher prices for an item with the exact resistances they want.  As you yourself noted, the prices for these is higher because fewer of them might exist, and that's exactly the point - that's our market and you're keeping us from being able to see it, so we now have to go back to doing manual searches to price those rings/amulets/belts.  Also this is not just league start because new people join a league every day.  When I came back to the game after a few years away I started in Standard then switched to League part of the way through the league.

Regarding (b): even for players who do know crafting - later in the league if they have 10ex in their bank and want to start a new character and they're being lazy ... if they want to pay an extra 20c to find an item that already has the specific 3 mods they want at the levels they want, who are you to tell us that we shouldn't be willing to sell to them, or that we can't price the item appropriately for that?

And lastly ... Harvest?  Harvest hasn't even been brought up into this conversation except by you.  If someone is looking to buy an item with all 3 resists Harvest tool is useless to them because Harvest swaps one res for another, and would still leave a third res empty.

**Comment by hejz0r** [comment]
@flapanther33781 : exactly, there are much more people with money who don't know basics of the game, and they want to spend money on "shity" 3 ress boots for example ..

**Comment by flapanther33781** [comment]
I mean, I understand his point that there are better ways to gear your character.  But I think I had built maybe 5-6 characters and gone through multiple leagues before I learned that.  Maybe he learned it quickly, or forgot how long it took him to learn, or he's underestimating the number of people who either haven't learned, never want to learn, or are lazy and willing to pay.

And he'd be right in saying there's no guarantee that our items will sell at that higher price.  We may list it and then bring it down over time (as everyone does anyway), but the point is that they might, and sometimes do, but we no longer know what the higher end of the range is in order to give it that initial price without having to go back and do manual searches for all three mods.

**Comment by hejz0r** [comment]
I wish he'll give us back this feature :(

**Comment by SlashGeeGee** [comment]
hmmm.. all my Res is at 45 each. Damn.. i really need a ring with triple 30 res but I guess I'll just settle for one with double 45 coz Snos said its "the same" lmao

**Comment by emailsbecker** [comment]
Got an email notification from the last update, was reading back through the comments.  I didn't point this out earlier, but:

> It's not more accurate. Sorry but you was doing it wrong. Exact ele res type doesn't matter. Just as an example, on your item buyer can easily change Crafted res and also Explicit lightning res with Harvest craft.

Harvest craft can be 10-20c, which could cost more than the item is worth for low-end accessories, and it would also be more time-consuming because it would require a buyer to buy a ring and then find a crafter to buy the craft from rather than just buying the ring with the res they want already on it.  This time-saving element is exactly why some rings might sell for more based on exactly which res mods it has rather than forcing the buyer to go through the steps that you think are "correct".

I still think this feature is worthwhile to users.  At a certain point I feel like just saying look, re-enable the feature, and if you go another year without understanding it, or never understand it, who cares.  We understand it, and we're asking for it.  And you already had it coded, so it's not like it would take you a lot of extra work.  I still don't understand why you took it away to begin with.

**Comment by flapanther33781** [comment]
Got an email notification from the last update, was reading back through the comments. I didn't point this out earlier, but:

> It's not more accurate. Sorry but you was doing it wrong. Exact ele res type doesn't matter. Just as an example, on your item buyer can easily change Crafted res and also Explicit lightning res with Harvest craft.

Harvest craft can be 10-20c, which could cost more than the item is worth for low-end accessories, and it would also be more time-consuming because it would require a buyer to buy a ring and then find a crafter to buy the craft from rather than just buying the ring with the res they want already on it. This time-saving element is exactly why some rings might sell for more based on exactly which res mods it has rather than forcing the buyer to go through the steps that you think are "correct".

I still think this feature is worthwhile to users. At a certain point I feel like just saying look, re-enable the feature, and if you go another year without understanding it, or never understand it, who cares. We understand it, and we're asking for it. And you already had it coded, so it's not like it would take you a lot of extra work. I still don't understand why you took it away to begin with.

**Comment by SnosMe** [comment]
> Harvest craft can be 10-20c

You don't buy them, they're common af. You also don't have anything valuable to store in station, so space is not an issue too.


1. Look here I'm searching for Total res and have **30 results** to chose from, starting from **70chaos**.

![image](https://user-images.githubusercontent.com/4292308/170003753-71c18120-9832-4181-9942-d57029144af6.png)

I'll buy this one for 1EX swap Fire to Cold, and craft something on open prefix.
![image](https://user-images.githubusercontent.com/4292308/170003849-f3e98d74-efa3-40b8-b630-5af06c9887e0.png)

---

2. Now I'm dumb and will search for exact wanted res
![image](https://user-images.githubusercontent.com/4292308/170003983-afbecac9-a90a-40cc-9f0a-55b972175fbe.png)

**Only one result** for 3EX with shitty T7 mana prefix
![image](https://user-images.githubusercontent.com/4292308/170004047-cea0d81d-f84e-41a8-a49f-36a90a92373f.png)

---

We may also have a disconnect of what items you're intended to price check.
- I'm tweaking the app towards price checking mid/end-game items, where it has some rare high tier mod with a more common mods wanted by everyone, by relaxing this common mods you find more results (this also helps with HC and private leagues where APT works too).
- And your examples are more about early game where you want to rise price from 5c to 30c, on item that doesn't have rare mod but only common ones, and you expect **someone** to search by exact values of them. Who will buy these overpriced items? Scamming **new** innocent **players**, welcoming beginner-friendly community indeed.
//Current decision is not based on the following text but I hope with introduction of Loot 2.0 you will find much more items on the ground with common mods that are good rolled, therefore much fewer people to buy items in early mapping, more good rolled ones on trade.

Also don't forget about Rog, he is literary shitting items with T1 common mods, very accessible.

**Comment by SlashGeeGee** [comment]
you just proved our point bro..  majority of the players dont know how to search like you did and will search exactly what they need. that said, if your tool allows searching separate resistance again, I would've priced mine like 2.5ex and someone would've bought it eventually. let us search the way we want to search and not the way you think we should.

**Comment by flapanther33781** [comment]
I saw this in the release notes for the most recent version: "Attribute Mods are now hidden rather than removed completely."  If this is restoring the functionality we've been discussing here, thank you, I appreciate it, and I'm sure others will as well.

That said, if it is what we've been discussing here, it's not working for me.  I've confirmed I'm running v3.18.10003 but I'm still only seeing 1 res mod.  Also, I'm a little confused about the Hidden slider and the Pseudo and Base Item tabs.  It seems like these tabs perform the same function as the Hidden slider.  But in neither case do I see more than one resist.
